coremltools/converters/mil/mil/ops/defs/control_flow.py

```diff
@@ -499,6 +626,27 @@ def type_inference(self):
 
 @register_op(doc_str="")
 class list_scatter(Operation):
+    """
+    Scatter ``values`` to ``ls`` at locations ``indices``.
+    
+    Parameters
+    ----------
+    ls: List[*] (Required)
+    
+    indices: <num_updates,i32> (Required)
```

Comment:
indices is a tensor. Shall we write it like `tensor<num_updates, i32>`?

Reply:
OK, fixed in next commit.